tests/common/jobs/test_elastic.py

```diff
@@ -54,3 +57,137 @@ def _get_strains(structure, sym_reduce):
     deformations = response[job.uuid][1].output
 
     return [d.green_lagrange_strain for d in deformations]
+
```

Comment:
Might be good to add a test for the whole flow or at least nake sure that the parameter won't be removed.

Reply:
Adding a test for the whole workflow is a bit problematic because it is difficult to consistently trigger the failure of one of the pertrubations. I can manually generate a test by explicitly killing vasp for one of the perturbations, but if the tests will need to be regenerated in the future it may be annoying.
It would be easier if I could modify the INCAR of only one of the perturbations, but it seems to not be possible because `update_kwargs` is not applied to dynamically generated jobs (see https://github.com/materialsproject/jobflow/issues/588).
I thought these tests will be enough, since all the logic stays in the fit job, but if you think it is needed I can try to generate the workflow test by killing the job. Or maybe there is a better way?

Reply:
Ah, I see. This sounds like too much work.

I think the fitting part is well tested. I was rather thinking about testing that a parameter change really arrives in the fit function

Reply:
If it was part of the output schema, it would be easy. 😅

Reply:
I think this is the only missing point. As this is a rather small but still important pull request, I would in this case merge it.
Does it make sense to add the max_* to the output schema and just test if it arrives there if you set it in the wf?

Reply:
Sorry, I think I had partially misunderstood your previous comment. I could add `max_failed_deformations` to the output document, but I am not sure if it is really worth to have that information just to make the test. I will try if I can quickly make a test by explicitly killing one of the deformation jobs, if it gets too complicated I will add `max_failed_deformations`.

Reply:
I managed to add a test with the flow failure triggered by the `max_failed_deformations` being set to 0.

Reply:
Awesome. Thank you!